content/md/en/docs/reference/how-to-guides/offchain-workers/offchain-transactions.md

```diff
@@ -348,7 +351,10 @@ Unsigned transactions always represent a potential attack vector that a maliciou
 
    In this example, users can call the on-chain `extrinsic1` function without a signature, but not any other extrinsic.
 
-   To see a full example of how `ValidateUnsigned` is implemented in a pallet, refer to [`pallet-example-offchain-worker` in **Substrate**](https://github.com/paritytech/substrate/blob/master/frame/examples/offchain-worker/src/lib.rs#L301-L329).
+   For an example of how `ValidateUnsigned` is implemented in a pallet, see the code for the [offchain-worker](https://github.com/paritytech/substrate/blob/master/frame/examples/offchain-worker/src/lib.rs#L301-L329).
+	 However, you should note that the code in the example only checks whether the provided `signature` is valid for the signed `payload` key. 
+	 This check doesn't validate whether the signer is an offchain worker or authorized to call the specified function.
+	 This simple check wouldn't prevent an unauthorized actor from using the signed payload to modify state.
```

Comment:
These lines are talking about Unsigned Tx w/ Signed Payloads but in the section of Unsigned Tx.

They should be moved to line `511`